package-lock.json

```diff
@@ -22,6 +22,7 @@
         "@clr/icons": "13.0.2",
         "@clr/ui": "^17.0.0",
         "@kolkov/angular-editor": "2.1.0",
+        "@ngx-translate/core": "15.0.0",
```

Comment:
pls also execute "npm i" to update the lock file

Reply:
Done